packages/jest-cli/src/TestRunner.js

```diff
@@ -228,17 +232,17 @@ class TestRunner {
 
     // Send test suites to workers continuously instead of all at once to track
     // the start time of individual tests.
-    const runTestInWorker = ({config, path}) =>
+    const runTestInWorker = test =>
```

Comment:
Flow isn't asking me to annotate this which means it infers the proper type. Why should I change it? I don't want to add superfluous types.